AEPTarget/Sources/Target.swift

```diff
@@ -580,6 +580,24 @@ public class Target: NSObject, Extension {
         return "\(coreVersion)+\(TargetConstants.EXTENSION_VERSION)"
     }
 
+    private func getSdkInfo(eventHubData: [String: Any]?) -> String {
+        let sdkBase = TargetConstants.HEADER_X_EXC_SDK_BASE_TARGET_MOBILE_IOS
+
+        guard let eventHubData = eventHubData else {
+            return sdkBase
+        }
+
+        guard let wrapper = eventHubData[TargetConstants.EventHub.SharedState.Keys.WRAPPER] as? [String: Any],
+              let wrapperFriendlyName = wrapper[TargetConstants.EventHub.SharedState.Keys.WRAPPER_FRIENDLY_NAME] as? String,
+              wrapperFriendlyName != "None"
+        else {
+            return sdkBase
+        }
+
+        // sdkInfo is of the format AdobeTargetMobile-iOS<-wrapperFriendlyName>
```

Comment:
i'd change this to `AdobeTargetMobile-iOS-<wrapperFriendlyName>`

Reply:
Wouldn't this imply if there is no wrapper, it would be `AdobeTargetMobile-iOS-`?

Reply:
Thank you! I got it now.